api/src/main/java/io/strimzi/api/kafka/model/authentication/KafkaClientAuthenticationOAuth.java

```diff
@@ -38,6 +38,8 @@ public class KafkaClientAuthenticationOAuth extends KafkaClientAuthentication {
     private String tokenEndpointUri;
     private Integer connectTimeoutSeconds;
     private Integer readTimeoutSeconds;
+    private Integer httpRetries;
+    private Integer httpRetryPauseMillis;
```

Comment:
I think we would normally use the `Ms` suffix. I.e. `httpRetryPauseMs`. For example `expireAfterMs` in OPA authorization. The `ms` suffix is also used in Kafka configs.

Reply:
I didn't notice, but sure we can have Ms. Intuitively it's just automatic to me that if we have SomethingSeconds, and not SomethingS, then we also have SomethingMillis and not SomethingMs. But I can change it since it's established pattern.
I don't really want to change it in OAuth lib though.

Reply:
I'm not so concerned about the OAuth lib, so feel free to keep it there and change it here. I think the `seconds` versus `ms` is commonly used elsewhere as well ... such as in Kafka itself. I guess `Millis` is not clear, `Milliseconds` is too long, and using just `S` for seconds` would be confusing.